synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -1045,6 +1055,41 @@
         "value": "CD8+ T-Cells",
         "description": "",
         "source": ""
+      },
+      {
+        "value": "CRISPR arachnoid",
```

Comment:
Yes- i think that's the best solution. I had added these before I looked at your recent pull request.